tasks/gotest.py

```diff
@@ -842,6 +833,12 @@ def format_packages(ctx: Context, impacted_packages: set[str], build_tags: list[
         for module in modules_to_test:
             print(f"- {module}: {modules_to_test[module].test_targets}")
 
+    # We need to make sure the CLI length is not too long
+    packages = compute_gotestsum_cli_args(modules_to_test.values())
+    if len(packages) > WINDOWS_MAX_CLI_LENGTH:
```

Comment:
```suggestion
    if sys.platform "win32" and len(packages) > WINDOWS_MAX_CLI_LENGTH:
```
`sys` is already included in this file. I suppose we only need to limit this on windows (which is unfortunate because it's the slowest environment 😭 )